.github/workflows/lint.yaml

```diff
@@ -15,18 +15,21 @@ jobs:
       - name: Check out the repository
         uses: actions/checkout@v4
 
-      - name: Set up Python
-        uses: actions/setup-python@v5
+      - name: Install the latest version of uv and set the python version to 3.13
+        uses: astral-sh/setup-uv@v6
         with:
-          python-version: '3.11'
+          python-version: 3.13
+          activate-environment: true
 
       - name: Install pre-commit
-        run: pip install pre-commit
+        run: uv sync --extra lint
 
       - uses: actions/cache@v4
         with:
           path: ~/.cache/pre-commit/
           key: pre-commit-4|${{ env.pythonLocation }}|${{ hashFiles('.pre-commit-config.yaml') }}
 
       - name: pre-commit
-        run: pre-commit run --all-files
+        env:
+          FORCE_PRE_COMMIT_UV_PATCH: 1
```

Comment:
Can you add a comment or something to explain why this is necessary? I see it listed in here https://github.com/tox-dev/pre-commit-uv but I'm not an expert with pre-commit and don't really understand the explanation in that repo 😬